### ISSUE TITLE
Autocomplete Field - autocomplete instructions were not available for some screen reading software

### ISSUE THREAD

**Issue by JuliaSang** [w3c/w3c-website#4]
Raised by the Digital Accessibility Centre as part of their accessibility testing: 

Autocomplete
The autocomplete instructions were not available for some screen reading software.

WCAG Reference:
Usability
Issue ID: DAC-Autocomplete

URL: https://w3c-dev.studio24.dev/forms.html 

Key audible feedback from the autocomplete was missing for JAWS users using Edge Chromium and VoiceOver users. No information on how to locate the options were present, and there was no indication that options has filtered.

VoiceOver users were not informed on how to interact with the component (such as by using Explore by touch).

Current code ref(s):
`<input aria-expanded="false" aria-owns="destination__listbox" aria-autocomplete="both" autocomplete="off" class="autocomplete__input autocomplete__input--show-all-values" id="destination" name="" placeholder="" type="text" role="combobox" aria-describedby="destination__assistiveHint">`
 

Screen reader comments:
JAWS: browsing with Microsoft Edge Chromium I was not advised of how to locate the options using any method of navigation. Once I entered characters, I was not informed that options were filtering and in a non-testing environment would not have been aware that I could filter content.

VoiceOver: I was advised that options were filtering once I entered characters, but no prompt advised me that I needed to explore by touch. As a highly experienced screen reader analyst I was aware of how to locate the options, but less experienced users might be unaware that they must explore by touch to locate the content.

TalkBack: I was given a prompt when in context of how to use the auto complete feature and could select an option.

Ensuring that screen reader users are advised on how to navigate to and select an option with all software will avoid any difficulty.”

Issue consistent for the following pages:
The W3C team
Journey 2 step 2
The W3C team | [W3C Redesign Prototypes (studio24.dev)](https://w3c-dev.studio24.dev/listing-people/index.html)
Members
Journey 2 step 3
[https://w3c-dev.studio24.dev/listing-members/index.html](https://w3c-dev.studio24.dev/listing-members/index.html)

**Solution:**
Ensure the autocomplete instructions on how to interact with the feature and its functionality when filtering options is available for all users of assistive technology. 

Please refer to the [GOV.UK Design System for accessible autocomplete examples](https://alphagov.github.io/accessible-autocomplete/examples/).
Note: We have not tested the GOV.UK Design System features in Microsoft Edge as this is not part of the testing matrix for GOV.UK Design System so were unaware that this was an issue in this browser and appears to be browser specific. Have moved to Usability section as does not prevent users from making a selection.

**Comment by JuliaSang** [comment]
This is something we have decided we will not resolve in this project phase for the following reasons: 

- It is a usability issue, not an accessibility issue (i.e. it works, just not as well as it should do in Edge) 
- It is only occurring on the Microsoft Edge browser
- It uses the same autocomplete as available on the gov.uk design system, which is showing the same bug. The example DAC refer us to is this one, although they concede they hadn't tested it previously on Edge, so hadn't spotted the issue
- We don't have time to rebuild the field and test, but as it still functions, we see it being a low priority anyway
- We've raised it as a bug with gov.uk in github, if they find a solution it can be brought over to this project in time.

**Comment by NicolaSaunders** [comment]
Closing this issue based on the previous comment
